utils/firestore.js

```diff
@@ -996,62 +998,36 @@ const searchSpecimen = async (masterSpecimenId, siteCode) => {
 }
 
 const searchShipments = async (siteCode) => {
-    const snapshot = await db.collection('biospecimen').where('827220437', '==', siteCode).get();
+    const { reportMissingTube, submitShipmentFlag, no } = fieldMapping;
+    const healthCareProvider = fieldMapping.healthCareProvider.toString();
+    const tubesBagsCidKeys = swapObjKeysAndValues(tubesBagsCids);
+    const snapshot = await db.collection('biospecimen').where(healthCareProvider, '==', siteCode).get();
+
     if(snapshot.size !== 0){
-        //
-        return snapshot.docs.filter(document => {
-            
-            let data = document.data();
+        let shipmentData = [];
+
+        for(let documents of snapshot.docs) {
```

Comment:
```suggestion
        for(let document of snapshot.docs) {
```

Reply:
Nice catch, I should've made documents singular.